clusterloader2/pkg/measurement/common/apiserverlatency.go

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
Please rename this file to apilatency.go (as apiserver latency can mean a thousand things).

Reply:
Or even better apicalllatency.go.

Reply:
done